contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -956,6 +956,7 @@ export const ContentNode = new Resource({
           lft,
           source_channel_id: node.channel_id,
           source_node_id: node.node_id,
+          channel_id: parentNode.channel_id,
```

Comment:
@rtibbles What do you think of this change? I noticed we weren't overriding `channel_id`.

Reply:
Yeah, that probably seems wise. It would get overridden on refresh, but good to have it consistent in the interim!